spec/e2e/has_many_spec.rb

```diff
@@ -387,13 +387,29 @@
 
   describe 'association "getter" options' do
     before do
+      Person.has_many :out, :best_friend, model_class: 'Person', type: nil, labels: false
       node.knows << friend1
       friend1.knows << friend2
     end
 
     it 'allows passing only a hash of options when naming node/rel is not needed' do
       expect(node.knows(rel_length: :any).to_a).to match_array([friend1, friend2])
     end
+
+    it 'honors default options' do
+      expect(node.knows.instance_variable_get('@query_proxy').instance_variable_get('@association_labels')).to eq(nil)
```

Comment:
I wonder if this could be testing by calling `to_cypher` on the `QueryProxy` and doing a `MATCH` on the string perhaps